dxr/hgext/previous_revisions.py

```diff
@@ -0,0 +1,35 @@
+"""
+Mercurial extension that prints the last node in which each tracked file changed.
+"""
+# Copyright (C) 2015,  Peter Elmers
```

Comment:
You probably want to assign copyright to Mozilla Foundation, as I believe it makes license reassignment easier (something something about needing everyone's permissions to relicense and that's much easier when a non-human entity holds copyright).